frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -42,16 +43,12 @@ export const FormComponentConfig = ({
     optionsId,
     hasCustomFileEndings,
     validFileEndings,
-    children,
```

Comment:
`children` was extracted here to ensure it is not rendered as a config property, and that should still be the case. Children is set by dragging components into the container component, it is not set in the config panel.

Reply:
Ah, I only saw that it was not rendered in the list of unsupported properties so I figured it worked, but when I take a closer look now I see that the prop is rendered as a title in the config-panel 😅 Will add it back!

![Skjermbilde 2024-02-15 kl  08 46 38](https://github.com/Altinn/altinn-studio/assets/71079896/e45d8fff-ee8d-4db6-b239-1a02ab04746e)